doc/setup.md

```diff
@@ -17,36 +17,31 @@ When installing SkyPortal on Debian-based systems, 2 additional packages are req
 - libcurl4-gnutls-dev
 - libgnutls28-dev
 
-## Package Managers
-Install `uv` and `Bun`:
-```
-# Install uv (faster Python package manager)
-curl -LsSf https://astral.sh/uv/install.sh | sh
+## Development Environment Setup
+SkyPortal is a full-stack application with a Python backend and JavaScript frontend. You'll need to set up both environments.
 
-# Install bun (a javascript runtime, faster equivalent to `node+npm`)
-curl -fsSL https://bun.sh/install | bash
-```
-After installation, you'll need to add uv to your PATH.
-```angular2html
-source $HOME/.cargo/env # for bash, zsh, or sh shells
-source $HOME/.cargo/env.fish # for fish shell
-```
-For bun, reload your shell:
+### Backend Environment (Python)
+We use [uv](https://docs.astral.sh/uv/) for Python package management and virtual environment creation. It's a faster alternative to pip and virtualenv.
+To install uv, run
 ```angular2html
-# For bash
-exec /bin/bash
-
-# For zsh
-exec /bin/zsh
+# macOS/ Linux
+curl -LsSf https://astral.sh/uv/install.sh | sh
 
-# For fish
-exec /bin/fish
+# Windows
+powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
 ```
-You can verify both installations succeeded by running:
+
+### Frontend Environment (JavaScript)
+For the frontend, we use [bun](https://bun.sh/docs), an all-in-one JavaScript toolkit that provides a fast JavaScript runtime (which replaces Node.js),
+a package manager (which replaces npm), and up to 4x faster startup times than Node.js. To install Bun, run
 ```angular2html
-uv --version
-bun --version
+# macOS / Linux
+curl -fsSL https://bun.sh/install | bash
+
+# Windows
+powershell -c "irm bun.sh/install.ps1|iex"
 ```
+Make sure to reload your shell or open a new terminal to access bun. You can also verify the install by running `bun --version`.
```

Comment:
Did you leave something like this for uv? Might be some git diff weirdness but while comparing with the previous version, it looks as if you only show that command for bun now